tests/blackbox/test_simple.py

```diff
@@ -33,6 +33,30 @@ async def test_connect_async(docker_client):
             assert results == [(2, 'two'), (3, 'three')]
 
 
+@mark.asyncio
+async def test_simultaneous_connections(docker_client):
+    async with SnowGlobeService.arun(docker_client) as service:
+        with connector.connect(**service.local_connection_kwargs()) as conn:
+            conn.cursor().execute('use database foo;')
+            conn.cursor().execute('create schema loolie')
+            conn.cursor().execute('use schema loolie')
+            conn.cursor().execute('create table bar (x int, y text)')
+            conn.cursor().execute("insert into bar values (1, 'one'), (2, 'two'), (3, 'three')")
+            # conn.cursor().execute('delete from bar where x = 10')
+            # results = conn.cursor().execute("select x, y from bar where y like 't%'").fetchall()
+            # assert results == [(2, 'two'), (3, 'three')]
```

Comment:
why?

Reply:
why commented or why forgot to delete commented lines?
commented because they are not needed the step is only needed to populate database and commit changes.

Reply:
then delete them